src/content/en/updates/2018/06/ar-for-the-web.md

```diff
@@ -0,0 +1,277 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Augmented reality allows placement and tracking of virtual objects in a real-world view.
+
+{# wf_updated_on: 2018-06-15 #}
+{# wf_published_on: 2018-06-14 #}
+{# wf_tags: immersive-web,webar #}
+{# wf_blink_components: Blink>WebVR #}
+{# wf_featured_image: /web/updates/images/2018/06/sunflowers.png #}
+{# wf_featured_snippet: Augmented reality allows placement and tracking of virtual objects in a real-world view. #}
+
+# Augmented reality for the web {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+In Chrome 67,
+[we announced](/web/updates/2018/05/welcome-to-immersive)
+the WebXR Device API for both augmented reality (AR) and virtual reality (VR),
+though only the VR features were enabled. VR is an experience based purely on
+what's in a computing device. AR on the other hand allows you to render virtual
+objects in the real world. To allow placement and tracking of those objects, we
+just added the
+[WebXR Hit Test API](https://github.com/immersive-web/hit-test/blob/master/explainer.md)
+to Chrome Canary, a new method that helps immersive web code place objects in
```

Comment:
Do you want to mention what version of Chrome that is?

Reply:
Apparently, no. The team intends to keep it canary only for next few cycles.

Reply:
Sounds good.